lib/ui/text.dart

```diff
@@ -90,7 +90,10 @@ enum TextAlign {
   right,
 
   /// Align the text in the center of the container
-  center
+  center,
+
+  /// Stretch each line of text to fill the width of the container
+  justify
```

Comment:
I presume you've checked that the numerical value of this enum matches the value we use in C++.